repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -16,10 +21,29 @@ class SetupTargetRepos(Actor):
     """
 
     name = 'setuptargetrepos'
-    consumes = (CustomTargetRepository, RepositoriesSetupTasks, RepositoriesMap, RepositoriesFacts, RepositoriesBlacklisted)
-    produces = (TargetRepositories,)
+    consumes = (CustomTargetRepository,
+                RepositoriesSetupTasks,
+                RepositoriesMap,
+                RepositoriesFacts,
+                RepositoriesBlacklisted,
+                RepositoriesUsage)
+    produces = (TargetRepositories, Report,)
     tags = (IPUWorkflowTag, ChecksPhaseTag)
 
+
+    def report_skipped_repos(self, repos, pkgs):
+        title = 'Some enabled RPM repositories are not compatible with Leapp'
+        summary_data = []
+        summary_data.append('Following repositories are unknown to Leapp:')
+        summary_data.extend(['- ' + r for r in repos])
+        summary_data.append('And the following installed packages from those repositories may not be upgraded:')
+        summary_data.extend(['- ' + p for p in pkgs])
+        summary = '\n'.join(summary_data)
+        reporting.report_generic(title=title, summary=summary, severity='low')
+        if True: #is_verbose():
+            api.show_message('\n'.join([title, summary]))
```

Comment:
I don't like using this function much, as the only thing it does is to print an message to stdout. It does not go to audit or logs.
Related: https://github.com/oamg/leapp/pull/497